### ISSUE TITLE
Add Finnish as language

### ISSUE THREAD

**Issue by alexgarciab** [stefangabos/world_countries#42]
Hi there, we want to use Country names in "Finnish" language. Would it be possible to add this language?

Regards

**Comment by stefangabos** [comment]
yes, but you will have to update the content on the [Wikipedia page](https://fi.wikipedia.org/wiki/Luettelo_ISO_3166_-standardin_koodeista) for that (we've just recently went through the same process adding names in Basque - see [here](https://github.com/stefangabos/world_countries/issues/38))
the following entries are missing

| name | alpha-2 | numeric | alpha-3 
|------|---------|---------|--------
Bonaire, Sint Eustatius and Saba | bq | 535 | bes
Curaçao | cw | 531 | cuw
Saint Barthélemy | bl | 652 | blm
Saint Martin (French part) | mf | 663 | maf
Sint Maarten (Dutch part) | sx | 534 | sxm
South Sudan | ss | 728 | ssd

**Comment by alexgarciab** [comment]
Done, 6 countries added to the Finnish Wikipedia page: https://fi.wikipedia.org/wiki/Luettelo_ISO_3166_-standardin_koodeista

Please, let me know when these countries are incorporated in this repo, so that we can update our code.

Thanks!

**Comment by stefangabos** [comment]
Done
I published a release and it is already available on [npmjs](https://www.npmjs.com/package/world_countries_lists) and should soon be also available on [jsdelivr](https://www.jsdelivr.com/package/npm/world_countries_lists)
Thanks!
